src/Components/Profile/MyPosts/myPosts.jsx

```diff
@@ -1,23 +1,24 @@
 import React from 'react';
 
+import { actionCreater } from '../../../redux/state';
+
 import ACTION_TYPES from '../../../actionTypes';
 import Post from './Post/post';
 
 import style from './myPosts.module.css';
 
+
```

Comment:
Consider removing this redundant empty line.

Reply:
Yeah.really redundant